utils/file/file.go

```diff
@@ -226,8 +226,11 @@ func DeleteFile(file string) error {
 func WriteChangeToFiles(ctx context.Context) error {
 	filesMap := ctx.Value(types.SERVICE_DETAIL).(types.ServiceDetail)
 	for file := range filesMap {
-		content, _ := yaml.Marshal(filesMap[file])
-		_, err := WriteToFile(file.(string), content)
+		content, err := yaml.Marshal(filesMap[file])
+		if err != nil {
+			log.Errorf("error occured while unmarshalling file from fileMap: %s", err)
```

Comment:
It will be good value add to print the value of fileMap variable here as well. That will help to know what was its value that caused error.

Reply:
I am not very comfortable doing that as printing the file map will make this error logs multi-line.

Reply:
@vipragupta : You have a valid concern. But, in my use-cases  I have always found printing  the data  causing  marshal/unmrashal error a valuable information to debug. I am just concerned here that if there is an error while marshalling, will that error  be informative enough to identify the  root cause? Since, we are not logging the filesMap here I  assume  in case  of error, we still have access to  filesMap content in that case. May be add some comment around what filesMap content is and where it is coming from.

Also, there is  a  typo in the error message. It should be `error occured while marshalling file from fileMap` as you are marshalling  and not  un-marshalling.